src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -14,6 +15,11 @@ public ComplexCommand()
 	{
 		this.commands = new ArrayList<>();
 	}
+
+	public ComplexCommand(ArrayList<DriverCommand> driverCommands) {
```

Comment:
Parameter type should point to the interface (and allow implementations other than ArrayList).